packages/design-system/src/__theme__/default/semantic.css

```diff
@@ -0,0 +1,128 @@
+/* Semantic tokens */
+:root {
+  /**
```

Comment:
Add a note saying that tokens from this file shouldn't be used, and category tokens should be used instead, explain that these tokens may change but the category tokens won't, and link to https://design-system.appsmith.com/?path=/docs/design-tokens--documentation for how to pick the correct token